Source: kloeri@exherbo.org
Upstream: No
Reason: Disable test requiring /tmp to be mounted with atime option

--- libio/Makefile.orig	2009-06-01 13:10:31.000000000 +0000
+++ libio/Makefile	2009-06-01 13:10:41.000000000 +0000
@@ -50,7 +50,7 @@
 tests = tst_swprintf tst_wprintf tst_swscanf tst_wscanf tst_getwc tst_putwc   \
 	tst_wprintf2 tst-widetext test-fmemopen tst-ext tst-ext2 tst-fopenloc \
 	tst-fgetws tst-ungetwc1 tst-ungetwc2 tst-swscanf tst-sscanf	      \
-	tst-mmap-setvbuf bug-ungetwc1 bug-ungetwc2 tst-atime tst-eof          \
+	tst-mmap-setvbuf bug-ungetwc1 bug-ungetwc2 tst-eof          \
 	tst-freopen bug-rewind bug-rewind2 bug-ungetc bug-fseek \
 	tst-mmap-eofsync tst-mmap-fflushsync bug-mmap-fflush \
 	tst-mmap2-eofsync tst-mmap-offend bug-fopena+ bug-wfflush \
